packages/backend/src/open_payments/payment/incoming/service.test.ts

```diff
@@ -124,7 +124,16 @@ describe('Incoming Payment Service', (): void => {
           walletAddressId,
           incomingAmount: {
             value: BigInt(123),
-            assetCode: asset.code.split('').reverse().join(''),
+            assetCode: String.fromCharCode(
+              ...asset.code.split('').map((letter) => {
+                let charCode = letter.charCodeAt(0)
+                if (charCode < 91) {
```

Comment:
nice solution. I think we need to check the lower bound though (65), so we don't go under 'A'. (maybe we always decrease and then wrap around to 'Z' (90)?)

Reply:
Feel free to completely ignore this, but I had fun thinking about Max's suggestion and came up with this.
 
```ts
assetCode: String.fromCharCode(
  ...asset.code
    .split('')
    .map((letter) => ((letter.charCodeAt(0) + 1 - 65) % 26) + 65)
),
```

Reply:
I like it. I'll just go ahead and steal that...